static/app/views/onboarding/targetedOnboarding/setupDocs.tsx

```diff
@@ -262,11 +362,80 @@ function SetupDocs({organization, projects, search}: Props) {
           }}
         />
       )}
+      {activeIntegration && (
+        <SetupIntegrationsFooter
+          organization={organization}
+          onClickSetupLater={() => {
+            trackAdvancedAnalyticsEvent(
+              'growth.onboarding_clicked_setup_integration_later',
+              {
+                organization,
+                integration: activeIntegration,
+                integration_index: activeIntegrationIndex,
+              }
+            );
+            // find the next uninstalled integration
+            const nextIntegration = selectedIntegrations.find(
+              (i, index) =>
+                !installedIntegrations.has(i) && index > activeIntegrationIndex
+            );
+            // check if we have an integration to set up next
+            if (nextIntegration) {
+              setNewActiveIntegration(nextIntegration);
+            } else {
+              // client state should always exist
+              clientState &&
+                setClientState({
+                  ...clientState,
+                  state: 'finished',
+                });
+              const nextUrl = `/organizations/${organization.slug}/issues/`;
+              browserHistory.push(nextUrl);
+            }
+          }}
+        />
+      )}
     </React.Fragment>
   );
 }
 
-export default withProjects(SetupDocs);
+type WrapperState = {
+  configurations: Integration[] | null;
+  information: {providers: IntegrationProvider[]} | null;
+} & AsyncComponent['state'];
+
+type WrapperProps = {
+  projects: Project[];
+  search: string;
+} & StepProps &
+  AsyncComponent['props'];
+
+class SetupDocsWrapper extends AsyncComponent<WrapperProps, WrapperState> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {slug, experiments} = this.props.organization;
+    // would be better to check the onboarding state for integrations instead of the experiment
+    // but we can't access a hook from a class component
+    if (!experiments.TargetedOnboardingIntegrationSelectExperiment) {
```

Comment:
How does returning `[]` here disable rendering integrations?
Do you also need to check the experiment for the # of stepper items and hiding the integration select page?

Reply:
@Neo-Zhixing Because the value of the state fields of `configurations` and `information` will be null because we never make an API call to those endpoints to load the data.

We will need the experiment value for the # of stepper items once we add the integration select page but that's outside this PR.